Source/Fuse.CameraRoll/iOS/iOSCameraRoll.uno

```diff
@@ -41,11 +41,28 @@ namespace Fuse.CameraRoll
 		[Foreign(Language.ObjC)]
 		static void SelectPictureInternal(Action<string> onComplete, Action<string> onFail)
 		@{
-			dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-				[[CameraRollHelper instance] selectPictureWithCompletionHandler:onComplete onFail:onFail];
-			});
+			PHAuthorizationStatus status = [PHPhotoLibrary authorizationStatus];
+			if (status == PHAuthorizationStatusNotDetermined)
+			{
+				[PHPhotoLibrary requestAuthorization:^(PHAuthorizationStatus status)
+				{
+					if (status == PHAuthorizationStatusAuthorized)
+					{
+						dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+							[[CameraRollHelper instance] selectPictureWithCompletionHandler:onComplete onFail:onFail];
+						});
+					} else {
+						onFail(@"User cancelled");
+					}
+				}];
+			} else if (status == PHAuthorizationStatusAuthorized)
+			{
+				dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+					[[CameraRollHelper instance] selectPictureWithCompletionHandler:onComplete onFail:onFail];
+				});
+			} else {
```

Comment:
I know we don't have a defined ObjC [coding-style defined](https://github.com/fuse-open/fuselibs/blob/master/Documentation/CodingStyle.md), but this seems even self-inconsistent.

I *think* most of the ObjC code I've seen uses K&R style braces. This isn't important at all for this PR, but perhaps we should consider standardizing this somehow?